packages/client/src/components/NewArrivals/NewArrivals.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useRef } from 'react';
+import Slider from 'react-slick';
+import {
+  Box,
+  Button,
+  IconButton,
+  Typography,
+  Link,
+  Container,
+} from '@mui/material';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import styled from '@emotion/styled';
+import { useNewArrivals } from './useNewArrivals';
+import './styles.css';
+
+const settings = {
+  className: 'center-slide',
+  centerMode: true,
+  infinite: true,
+  centerPadding: '370px',
+  slidesToShow: 1,
+  swipeToSlide: true,
+  slidesToScroll: 1,
+  dots: true,
+  arrows: false,
+  autoplay: false,
+  speed: 500,
+  autoplaySpeed: 3000,
+  cssEase: 'linear',
+};
+
+export function NewArrivals() {
+  const [newArrivals] = useNewArrivals();
+  const sliderRef = useRef();
+
+  const handleSliderPrev = () => {
+    sliderRef.current?.slickPrev();
+  };
+
+  const handleSliderNext = () => {
+    sliderRef.current?.slickNext();
+  };
+
+  return (
+    <MainContainer maxWidth="false" className="MAIN-CONTAINER">
```

Comment:
What is the use of class names here if you are using styled components? And why are class names in all caps?

Reply:
yeah because i was using Chrome inspect to arrange styles of the React-slick so i put name's to understand which DIV is which :)) i can remove while refactoring the project